src/main/java/seedu/ui/Ui.java

```diff
@@ -198,11 +197,11 @@ public void printListMessage(ArrayList<Ingredient> listOfIngredients, String sor
     public String sortTypeMessage(String sortType) {
         switch (sortType) {
         case "e":
-            return ("< Listing earliest [Expiry Date] first >");
+            return ("List sorted by expiry date:");
         case "r":
-            return ("< Listing Most Recently Added items last >");
+            return ("List sorted by how old add is:");
```

Comment:
This wording is a bit confusing. Can we change it to "List sorted by most recently added:"?